http/server.lua

```diff
@@ -1363,7 +1492,7 @@ local exports = {
             is_run  = false,
             stop    = httpd_stop,
             start   = httpd_start,
-            options = extend(default, options, true),
+            options = extend(default, options, false),
```

Comment:
Why is this has been changed?

Reply:
Some TLS options can be nil, but this boolean `raise` argument doesn't allow to have nil variables in `option` table.